main.py

```diff
@@ -91,9 +97,20 @@ def __init__(self, *args, **kwargs):
             # sanitize input string headed to command executor
             sanitized_message = sanitize(message)
             log.debug(f"Sanitized {sanitized_message}, Original: {message}")
-            await rat_command.trigger(message=sanitized_message,
-                                      sender=user,
-                                      channel=channel)
+            try:
+                await rat_command.trigger(message=sanitized_message,
+                                          sender=user,
+                                          channel=channel)
+            except CommandNotFoundException:
+                # command was not found
+                log.debug("Command invoked from \"{message}\" not found.")
+
+            except Exception as ex:
```

Comment:
I might be blind, but does the exception get logged at all? ("raise" or "log.exception(ex)" )

Reply:
see line 110
```py
            except Exception as ex:
                ex_uuid = uuid4()
                log.exception(ex_uuid)  # Hi there BeepBeat!
                error_message = graceful_errors.make_graceful(ex, ex_uuid)
                # and report it to the user
                await self.message(channel, error_message)
```